src/main/java/frc/robot/commandGroup/PickUpCube.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.commandGroup;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.arm.commands.SetArmsPosition;
+import frc.robot.subsystems.gripper.Gripper;
+import frc.robot.subsystems.intake.Command.Feed;
+
+public class PickUpCube extends SequentialCommandGroup {
+    public PickUpCube(Gripper gripper, double intakePower, Translation2d armPosition) {
```

Comment:
Why do the arm position and intake power need to be passed as a parameter?
On the other hand, the arm should be passed as a parameter.

Reply:
Why did you resolve this conversation?